Dockerfile

```diff
@@ -1,10 +1,10 @@
 
-FROM openshift/origin-release:golang-1.10 as builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
```

Comment:
Is VPN access required for this? We need ensure this works if anyone tried using the operator. Moreover I see that the OLM [Dockerfile](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Dockerfile#L1) does not do this and they have gone through the OSBS process.

Reply:
I did just test this and it does work without VPN. Moreover, I believe this is a standard that the release team is trying to enforce. Take a look at this documentation for more detail:

https://mojo.redhat.com/docs/DOC-1179058#jive_content_id_Request_Addition_to_ART_Build_System

There are several operator repos that are using these versions of the base image. It seems likely that the OLM source code is just not following the standard.

Reply:
For example:

https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/Dockerfile.rhel7#L1

https://github.com/openshift/cluster-samples-operator/blob/master/Dockerfile#L1

https://github.com/openshift/cluster-openshift-apiserver-operator/blob/master/Dockerfile.rhel7#L1

https://github.com/openshift/cluster-ingress-operator/blob/master/Dockerfile.rhel7#L1